<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Set_interval</class>
 <widget class="QDialog" name="Set_interval">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/tubiao/guo.png);</string>
  </property>
  <widget class="QPushButton" name="P_sure">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/tubiao/10.png);
color: rgb(255, 255, 255);
font: 75 18pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="P_quit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>360</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/tubiao/10.png);
color: rgb(255, 255, 255);
font: 75 18pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="Text_up" name="P_textedit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>111</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 36pt &quot;Arial&quot;;</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:36pt; font-weight:80; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:72pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="S_word">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>351</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>29</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>请输入时间间隔：</string>
   </property>
  </widget>
  <widget class="QLabel" name="S_h">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>160</y>
     <width>41</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>h</string>
   </property>
  </widget>
  <widget class="Text_up" name="P_textedit1">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>160</y>
     <width>121</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 36pt &quot;Arial&quot;;</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:36pt; font-weight:80; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:72pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="S_min">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>170</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>min</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Text_up</class>
   <extends>QTextEdit</extends>
   <header location="global">text_up.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
